CHANGELOG.md

```diff
@@ -12,7 +12,10 @@
 
 ### Enhancements
 
-* None.
+* Add new `ignore_identity_closures` parameter to `prefer_key_paths` rule to skip conversion of identity closures 
+  (`{ $0 }`) to identity key paths (`\self`).
+  Add a small note to the rule description stating that identity key path conversion is Swift 6+ only.  
```

Comment:
Let's have the note here as well instead of a reference to the note:

```suggestion
  Note that identity key paths are only support from Swift 6 on, hence this option
  will be implicitly ignored/set to `true` when SwiftLint detects a Swift <6 compiler 
  to avoid causing compilation errors.  
```